src/main/java/net/snowflake/ingest/utils/ErrorCode.java

```diff
@@ -26,7 +26,7 @@ public enum ErrorCode {
   INVALID_ENCRYPTED_KEY("0018"),
   INVALID_DATA_IN_CHUNK("0019"),
   IO_ERROR("0020"),
```

Comment:
It looks like error code actually gets exposed to customers via SFException.getErrorCode().
Thus this is a breaking change, please don't rename an error code.

@sfc-gh-tzhang pl confirm?

Reply:
yeah, I tagged you in another thread, please don't modify existing error codes, just add new ones, see https://github.com/snowflakedb/snowflake-ingest-java/pull/794#discussion_r1679676642

Reply:
Reverted change.